src/api/code-challenge/content-types/code-challenge/utils.ts

```diff
@@ -0,0 +1,156 @@
+// Import filepaths changed from original
+import { removeJSComments } from "../code-challenge/curriculum-helpers";
+import { getErrorMessage } from "../code-challenge/general";
+
+type consoleLogOverride = (data: unknown[]) => void;
+
+export const overrideConsoleLog = (override: consoleLogOverride) => {
+  // @ts-expect-error will fix later
+  console.standardLog = console.log;
+  console.log = (...data: unknown[]) => {
+    override(data);
+    // @ts-expect-error will fix later
+    console.standardLog.apply(console, data); // eslint-disable-line prefer-spread
+  };
+};
+
+export const restoreConsoleLog = () => {
+  // @ts-expect-error will fix later
+  if (!console.standardLog) {
+    throw new Error(
+      "Attempted to restore console.log but it has never been overwritten"
+    );
+  }
+  // @ts-expect-error will fix later
+  console.log = console.standardLog;
+};
+
+const getCodeEvaluationHelpers = (
+  logs: Array<Array<unknown>>,
+  codeString: string
+) => {
+  const helpers = {
+    // right now this only works with primitives
+    wasLoggedToConsole: (val: unknown) => {
+      return logs.some((logGroup) => logGroup.some((log) => log === val));
+    },
+    wasFunctionInvoked: (functionName: string) => {
+      return codeString.trim().includes(`${functionName}()`);
+    },
+    /**
+     * Evaluates a condition for each line of the provided code.
+     * Upon finding a line that passes it returns that line's index
+     */
+    findFirstPassingLineForCondition: ({
+      condition,
+    }: {
+      condition: string;
+    }) => {
+      debugger;
```

Comment:
remove

Reply:
Can't, it is currenty used in test-evaluators.ts
```
import {
  evaluateWithContext,
  getCode,
  getEvaluationContext,
  overrideConsoleLog,
  restoreConsoleLog,
} from "../code-challenge/utils";
```